tests/drivers/can/timing/src/main.c

```diff
@@ -215,11 +206,16 @@ static void test_timing_values(const struct device *dev, const struct can_timing
 ZTEST_USER(can_timing, test_timing)
 {
 	const struct device *const dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_canbus));
+	int count = 0;
 	int i;
 
 	for (i = 0; i < ARRAY_SIZE(can_timing_tests); i++) {
-		test_timing_values(dev, &can_timing_tests[i], false);
+		if (test_timing_values(dev, &can_timing_tests[i], false)) {
+			count++;
+		}
 	}
+
+	zassert_true(count > 0, "no bitrates supported");
```

Comment:
@str4t0m I reverted to implementing a "test counter" as suggested by you in https://github.com/zephyrproject-rtos/zephyr/pull/69533#discussion_r1505939431 here (and below). I thought I could have done something clever with generating test cases using the preprocessor, but our ztest framework currently does not fully support that.

Reply:
Thanks, yeah it should do it's job and maybe some ztest improvments will allow a better solution in the future.